### PR TITLE
Doubly ended queue in Python

### PR DESCRIPTION
**Pull request by shloka-gupta**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

(Yes)

### Description

(Since lists in python are dynamic in nature I have created a doubly ended queue using list in python)

### Checklist

- [Yes] I've read the contribution guidelines.
- [Yes] I've checked the issue list before deciding what to submit.
- [Yes] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Issue #7200)

### CODE REVIEW

**Comment by shloka-gupta** [comment]
@atarax665 / @ankitaggarwal23  could you kindly review this?

File: Python/ds/Deque_with_list.py [hunk 1]
```diff
@@ -0,0 +1,180 @@
+'''
+Doubly ended queue
+implementation using List in Python
```
Comment (ankitaggarwal23) [inline Python/ds/Deque_with_list.py:3]: - Add a beginner-friendly description at the top

File: Python/ds/Deque_with_list.py [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+
+try:
+    queue = []
+
+    # For the queue to be dynamic it has been put in a loop
+    # While the loop is true the user gets options:
+    # To insert at rear/front
+    # remove from front/rear
+    # display all elelements,reverse queue or exit
+
```
Comment (ankitaggarwal23) [inline Python/ds/Deque_with_list.py:14]: Remove this redundant part

File: Python/ds/Deque_with_list.py [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+    while True:
+        op = int(input('''
+Press-->
+1 to insert in rear
+2 to remove from front
+3 to insert in front
+4 to remove from rear
+5 to display values of queue
+6 to reverse the exisiting queue
+7 to exit '''))
```
Comment (ankitaggarwal23) [inline Python/ds/Deque_with_list.py:24]: Remove this redundant part

File: Python/ds/Deque_with_list.py [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+
+            if len(queue) == 0:
+                print("The queue is empty, insert values if required")
+            else:
+                ele = queue.pop(0)
+                print("Element removed is - ", ele)
+                print(queue)
+
+        elif op == 3:
+            ele = int(input("enter elem to insert "))
```
Comment (ankitaggarwal23) [inline Python/ds/Deque_with_list.py:41]: What is `elem`? Rewrite it to `Element`

File: Python/ds/Deque_with_list.py [hunk 5]
```diff
@@ -0,0 +1,180 @@
...
+            queue.append(ele)
+            print(queue)
+
+        elif op == 2:
+
+            if len(queue) == 0:
+                print("The queue is empty, insert values if required")
+            else:
+                ele = queue.pop(0)
```
Comment (ankitaggarwal23) [inline Python/ds/Deque_with_list.py:36]: Use proper naming conventions for declaring variables. `ele`

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]

**Review by ankitaggarwal23 — COMMENTED** [review]
Thank you for your valuable time.😊

**Pull request closed without merging**